doc/rst/developer/bestPractices/Valgrind.rst

```diff
@@ -46,3 +46,43 @@ The above options are needed because not all third-party libraries support
   reasons, unless ``CHPL_RE2_VALGRIND_SUPPORT=true`` is set at build time
 - GASNet support for ``valgrind`` is experimental at this time -- see 
   https://github.com/chapel-lang/chapel/issues/8544 for the current status
+
+LLVM Compatibility
+------------------
+
+Some versions of LLVM generate debugging information that some version
+of valgrind can't understand.  This can manifest as an error like the following:
```

Comment:
Small note about spelling, I think it should be "that some versions of valgrind"

Reply:
Hi @jabraham17 , oops, thanks.  Done.